theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,23 @@
+<%
+const query = site.queryPages;
+if (page.pathname !== '') {
+  const [queryPage] = query(({pathname}) => page.pathname === pathname);
```

Comment:
Detail/Suggestion: This holds all data about page, the query part in the beginning makes it sound like a verb, maybe we can name it just `page`?

Reply:
> 
> 
> Detail/Suggestion: This holds all data about page, the query part in the beginning makes it sound like a verb, maybe we can name it just `page`?

I think `currentPage` is better.

Reply:
Agree